components/brave_wallet_ui/page/wallet_page.tsx

```diff
@@ -26,13 +27,22 @@ import { ApiProxyContext } from '../common/context/api-proxy.context'
 // components
 import BraveCoreThemeProvider from '../../common/BraveCoreThemeProvider'
 import Container from './container'
-import { store, walletPageApiProxy } from './store'
+import { persistor, store, walletPageApiProxy } from './store'
+import LoadingSkeleton from '../components/shared/loading-skeleton'
 
 // style
 import walletDarkTheme from '../theme/wallet-dark'
 import walletLightTheme from '../theme/wallet-light'
 import 'emptykit.css'
 
+const onPageRefreshInitiated = async function (
```

Comment:
nit - non security blocking: We'll probably want to expose this as a button on some pages as well

Reply:
The primary thing I was thinking about here was refresh of portfolio data and NFT data, but I'm not even sure that aligns with this work so no reason to block on it.

Reply:
fine to ignore this one - I had a different idea about how we might be able to get a user initiated refresh